main/EventSystem.cpp

```diff
@@ -3670,6 +3672,14 @@ std::string CEventSystem::nValueToWording(const unsigned char dType, const unsig
 	else if (lstatus == "")
 	{
 		lstatus = sValue;
+	//OJO if lstatus  is still empty we use nValue for lstatus.
+        	lstatus = sValue;
+        	if (lstatus == "")
+        	{
+           		temp = nValue;
+           		ss << temp;
```

Comment:
no need to use temp
please declase the stringstream here, no need to put it outside this scope

you can just do

ss << (int) temp